pkg/scanner/local/scan.go

```diff
@@ -122,6 +113,28 @@ func (s Scanner) Scan(target string, imageID string, layerIDs []string, options
 	return results, imageDetail.OS, eosl, nil
 }
 
+func (s Scanner) addScanOSPkgResults(target string, imageDetail ftypes.ArtifactDetail, options types.ScanOptions) (report.Results, *ftypes.OS, bool, error) {
```

Comment:
Such a change would increase the PR size. It makes review difficult. Can we disable the linter once, revert this change, add ToDo, and fix it in the dedicated PR?